docs/integrations/connector-health.md

```diff
@@ -0,0 +1,63 @@
+# Connector Health
+
+Airbyte runs regular testing of all integrations to ensure we are able to properly interact with APIs/DBs/etc. Our goal
+is to quickly identify breaking changes.  
+
+This page shows the status of integration tests running for each connector on `master`. 
+Red status badges mean the last test failed. 
+Yellow status badges mean the last test passed, but at least one of the last ten tests failed. 
+Green status badges mean the last ten tests were all successful. 
+
+Clicking on a status will take you to a summary of past tests that link to the Github workflows executed for the test.
+
+The test status badges are cached and can take 5 minutes to update.
+
+## Sources
+
+| Source | Airbyte Certified | Status |
+| -- | -- | -- |
+| Braintree | | [![source-braintree-singer](https://img.shields.io/endpoint?url=https%3A%2F%2Fstatus-api.airbyte.io%2Ftests%2Fsummary%2Fsource-braintree-singer%2Fbadge.json)](https://status-api.airbyte.io/tests/summary/source-braintree-singer) |
```

Comment:
Actually after looking at this rendered, I'm going to put Airbyte Certified at the end.

Reply:
agreed